pyclam/criterion.py

```diff
@@ -263,146 +256,3 @@ def _select(root: Cluster, upper: float, lower: float, grace_depth: int) -> Set[
             inactive = {child for cluster in inactive for child in cluster.children}
 
         return selected
-
-
-def _find_parents_to_add(graph: Graph) -> Set[Cluster]:
-    # get sorted list of clusters
-    # guarantees that any siblings will be next to each other.
-    clusters: List[Cluster] = list(sorted(graph.clusters))
-    # find all sibling pairs
-    sibling_pairs: List[Tuple[Cluster, Cluster]] = [
-        (left, right) for left, right in zip(clusters[:-1], clusters[1:])
-        if (left.depth == right.depth
-            and left.name == right.name[:len(left.name)]
-            and {left, right}.issubset(graph.walkable_clusters))
-    ]
-
-    # get set of parents to add and set of clusters to keep
-    sibling_set: Set[Cluster] = set()
-    [sibling_set.update(pair) for pair in sibling_pairs]
-    remainder: Set[Cluster] = {cluster for cluster in graph.clusters if cluster not in sibling_set}
-    parents: Set[Cluster] = {left.parent for left, _ in sibling_pairs}
-
-    # Check if parent also does not subsume any cluster in remainder
-    # remove parent and restore children if so
-    removals: Set[Cluster] = {
-        parent for parent in parents
-        if any((
-            parent.radius >= d + c.radius
-            for c, d in parent.candidates.items()
-            if c in remainder
-        ))
-    }
-    parents -= removals
-    [remainder.update(cluster.children) for cluster in removals]
-
-    return remainder.union(parents)
-
-
-def _find_parents_to_remove(graph: Graph) -> Set[Cluster]:
-    # replace by children any cluster that:
-    #       subsumes an other cluster, and
-    #       is itself not subsumed by any other cluster.
-    removals: Set[Cluster] = {
-        cluster for cluster in graph.clusters
-        if (cluster.children
-            and cluster in graph.walkable_clusters
-            and graph.subsumed_edges[cluster])
-    }
-    remainder: Set[Cluster] = {cluster for cluster in graph.clusters if cluster not in removals}
-    additions: Set[Cluster] = set()
-    [additions.update(cluster.children) for cluster in removals]
-
-    return remainder.union(additions)
-
-
-class MinimizeSubsumed(GraphCriterion):
```

Comment:
Are we sure we want to get rid of these criteria?

Reply:
They are alive on a different branch where I'm trying to fix add and remove for Graph